### PR TITLE
Create opam principia-pl

### PR DESCRIPTION
**Pull request by LogicalAtomist**
Given that I do not use Linux, I am uncertain if submitting the package principia-pl is possible. Hopefully, there are no errors, but I was unable to follow the "test your new package" instructions on the opam-packaging page on Coq's website.

### CODE REVIEW

File: coq-principia-pl/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
+opam-version: "1.0"
```
Comment (clarus) [inline coq-principia-pl/opam:1]: This file should be at `released/packages/coq-principia-pl/coq-principia-pl.1.0.0/opam`.

Comment (clarus) [inline coq-principia-pl/opam:1]:
```suggestion
opam-version: "2.0"
```

File: coq-principia-pl/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+synopsis: "A Coq library for *Principia Mathematica*'s propositional logic" # One-line description
+description: """
+  This package checks all of *Principia*'s primitive axioms, inference rules, and definitions
+  against Coq's standard classical logic libraries; it also contains proofs that match the 
+  proof sketches given in *Principia* in that (a) the proofs cite every proof mentioned in 
+  *Principia*'s proof sketch and (b) nothing extraneous to *Principia*'s logic is used.
+""" # Longer description, can span several lines
```
Comment (clarus) [inline coq-principia-pl/opam:8]:
```suggestion
"""
```

File: coq-principia-pl/opam [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+homepage: "https://www.principiarewrite.com/"
+dev-repo: "https://github.com/LogicalAtomist/principia"
+bug-reports: "https://github.com/LogicalAtomist/principia-pl/issues"
+doc: "https://github.com/LogicalAtomist/principia/blob/master/PL.v"
+maintainer: "elkind@ualberta.ca"
+authors: [
+  "Landon D. C. Elkind"
+]
+license: "MIT" # Make sure this is reflected by a LICENSE file in your sources
```
Comment (clarus) [inline coq-principia-pl/opam:18]:
```suggestion
license: "MIT"
```

File: coq-principia-pl/opam [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+synopsis: "A Coq library for *Principia Mathematica*'s propositional logic" # One-line description
+description: """
+  This package checks all of *Principia*'s primitive axioms, inference rules, and definitions
+  against Coq's standard classical logic libraries; it also contains proofs that match the 
+  proof sketches given in *Principia* in that (a) the proofs cite every proof mentioned in 
+  *Principia*'s proof sketch and (b) nothing extraneous to *Principia*'s logic is used.
+""" # Longer description, can span several lines
+
+homepage: "https://www.principiarewrite.com/"
+dev-repo: "https://github.com/LogicalAtomist/principia"
```
Comment (clarus) [inline coq-principia-pl/opam:11]:
```suggestion
dev-repo: "git+https://github.com/LogicalAtomist/principia.git"
```

**Review by clarus — COMMENTED** [review]
Thanks for the submission. Some comments.

File: coq-principia-pl/opam (hunk 1 above)
Comment (LogicalAtomist) [reply coq-principia-pl/opam]: Thanks kindly for the corrections. I have moved the file to the suggested location.

**Comment by gares** [comment]
The file is in the wrong folder, it should be under released/packages/

File: coq-principia-pl/opam (hunk 1 above)
Comment (clarus) [reply coq-principia-pl/opam]: :+1:

**Comment by clarus** [comment]
@LogicalAtomist Hello, did you have the opportunity to make the https://github.com/LogicalAtomist/principia-pl repository public, so that it can be installed from opam?

**Comment by LogicalAtomist** [comment]
Greetings,

Thanks for asking about this. I see now what the issue is: the repo I have
been using is principia, not principia-pl. I have updated the branch name,
package name, and links in the package accordingly. This repo is public
already. Hopefully, that should fix the bug.

Yours Truly,
Landon

Landon D. C. Elkind
Killam Postdoctoral Fellow in Philosophy
University of Alberta (website <https://landondcelkind.com/>)
The *Principia* Rewrite Project <https://www.principiarewrite.com/>

**Comment by clarus** [comment]
Is this intentional that you closed the pull-request? Now you need to make a release for the version `1.0.0` in GitHub, as the error is the following:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - coq-principia.1.0.0:
            https://github.com/LogicalAtomist/principia/archive/1.0.0.tar.gz (curl: code 404 while downloading https://github.com/LogicalAtomist/principia/archive/1.0.0.tar.gz)
```

**Comment by LogicalAtomist** [comment]
No, sorry. I was trying to rename the repo folder in response to an earlier comment.

**Pull request closed without merging**